app/controllers/concerns/dradis/plugins/persistent_permissions.rb

```diff
@@ -0,0 +1,43 @@
+module Dradis
+  module Plugins
+    module PersistentPermissions
+      extend ActiveSupport::Concern
+
+      def update
+        @user = User.find(params[:id])
```

Comment:
Should this go through `User.testers`? to avoid Disabled and Contributors?

Reply:
Seems like a solid suggestion.

Reply:
I suppose even better for right now would be `authors` as we don't assign permissions to admin.